src/developer/web-api/tracker.md

```diff
@@ -2906,6 +2906,23 @@ scope is applied to the registration organisation unit of the TrackedEntity. Whe
 retrieving TrackedEntities, including specific program data, the organisation unit scope is applied
 to the Owner organisation unit.
 
+In scenarios where the capture scope and search scope do not overlap, the system will grant the
```

Comment:
Do we even need to expose the user to this level of detail? And in what scenarios?

As a user I would find it easier to understand that in x, y, z the union of capture and search scope are used. And in a, b, c only the capture or search scope is used.

Can we make such statements? 😅